### PR TITLE
Static Text Fix

### PR DESCRIPTION
**Pull request by MichaelChung123**
# Description
Fixing issue with PR 187. Issue caused with conditional rendering of Static Content.

## Types of changes
Moved the previous logic for the SimpleContent into a constructor block instead of the built in attach method.

## Checklist

- [X] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [X] I have checked that unit tests pass locally with my changes
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

## Further comments
Will be doing a QA screen in Test env to confirm it is safe before merging.

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://chefs-dev.apps.silver.devops.gov.bc.ca/pr-191

File: app/frontend/src/components/designer/FormViewer.vue [hunk 1]
```diff
@@ -129,6 +129,22 @@ export default {
...
     },
+    // Attaches attributes to <a> Link tags to open in a new tab.
+    attachAttributesToLinks() {
+      let simpleContentComponents = formioUtils.searchComponents(this.formSchema.components, {
+        type: 'simplecontent'
+      });
+      let advancedContent = formioUtils.searchComponents(this.formSchema.components,{
+        type: 'content'
+      });
+      console.log('simpleContentComponents: ', simpleContentComponents);
```
Comment (jujaga) [inline app/frontend/src/components/designer/FormViewer.vue:140]: I think you forgot to remove this console log.

File: app/frontend/src/components/designer/FormViewer.vue [hunk 2]
```diff
@@ -129,6 +129,22 @@ export default {
         this.loadingSubmission = false;
       }
     },
+    // Attaches attributes to <a> Link tags to open in a new tab.
+    attachAttributesToLinks() {
+      let simpleContentComponents = formioUtils.searchComponents(this.formSchema.components, {
```
Comment (jujaga) [inline app/frontend/src/components/designer/FormViewer.vue:134]: This should be a const - it's not being reassigned anywhere else in this function scope.

File: app/frontend/src/components/designer/FormViewer.vue [hunk 3]
```diff
@@ -127,6 +129,22 @@ export default {
...
+        type: 'simplecontent'
+      });
+      let advancedContent = formioUtils.searchComponents(this.formSchema.components,{
```
Comment (jujaga) [inline app/frontend/src/components/designer/FormViewer.vue:137]: This should be a const - it's not being reassigned anywhere else in this function scope.
Also needs a space after the comma and before the open curly brace.

**Review by jujaga — COMMENTED** [review]
Looks pretty good semantically. A future thought is that if we do end up needing to leverage formioUtils more heavily, it may be worth writing out these new functions as standalone JS modules so that it can be siloed away, imported and used as needed (but no need to do that yet since it'd only exist in this Vue component for the moment).

File: app/frontend/src/components/designer/FormViewer.vue (hunk 1 above)
Comment (MichaelChung123) [reply app/frontend/src/components/designer/FormViewer.vue:140]: Moving this fix into a new PR-196
https://github.com/bcgov/common-hosted-form-service/pull/196

File: app/frontend/src/components/designer/FormViewer.vue (hunk 2 above)
Comment (MichaelChung123) [reply app/frontend/src/components/designer/FormViewer.vue:134]: Moving this fix into a new PR-196
https://github.com/bcgov/common-hosted-form-service/pull/196

File: app/frontend/src/components/designer/FormViewer.vue (hunk 3 above)
Comment (MichaelChung123) [reply app/frontend/src/components/designer/FormViewer.vue:137]: Moving this fix into a new PR-196
https://github.com/bcgov/common-hosted-form-service/pull/196

**Pull request closed without merging**